index.html

```diff
@@ -787,36 +787,154 @@ <h4>`ThingLink`</h4>
+                MUST include a <code>WWW-Authenticate</code> header
+                and any other headers
+                describing the required authorizations using one of the following IANA-registered
+                HTTP Authentication Schemes: Basic, Bearer, or Digest.</span>
+                For details of the requirements, the IANA registry should be consulted for
+                each of the above authentication schemes.  
+                <span class="rfc2119-assertion" id="exploration-secboot-oauth2-flows">
+                If a "302 (Redirection)" response code is provided during security bootstrapping, 
+                the OAuth2 <code>code</code> flow MUST be used,
+                with access credentials eventually being represented with bearer tokens.
+                </span>
+                Note that the other OAuth2 flows supported in WoT Thing Description 1.1,
+                <code>client</code> and <code>device</code>, both expect the initial access
+                to be to the authentication server, not the final endpoint, and so cannot be
+                used in this case.
+                These requirements apply only to endpoints of exploration services that need
+                to support security bootstrapping, that is those that serve TDs,
+                not to other endpoints that might be
+                provided by the same exploration service.  
+                In particular, it applies
+                only to URLs that can be referenced by introduction mechanisms, not to (for example)
+                event subscription endpoints.
+                </li>
+                <li><span class="rfc2119-assertion" id="exploration-secboot-reqd-auth">Any
+                endpoint of an exploration service that can serve a TD MUST support
+                at least one of the security schemes supported by security bootstrapping.</span>
+                This is a logical consequence of the other requirements above.  This requirement
+                does not preclude the exploration endpoint supporting additional security schemes
+                more appropriate for a given use case (e.g. automation) if necessary.
+                </li>
+                </ul>
+                <p>
+                There are Security and Privacy Considerations in [[wot-architecture11]] 
+                and [[wot-thing-description11]] that are
+                relevant regarding when authentication is required for access
+                to TDs and the use of secure transport.  
+                In summary, secure transport (e.g. TLS) is required for public services and strongly recommended
+                even on private networks (even if there is no authentication requirement, to
+                protect the confidentiality of queries), and <code>nosec</code> should only be considered 
+                in limited circumstances when no Personally Identifiable Information is 
+                present or can be inferred.
```

Comment:
Same comment as before. Is "nosec" a defined security mechanism outside of the TD? If yes, a reference would be nice. If not:
```suggestion
                protect the confidentiality of queries), and serving requests without authentication and authorization should only be considered 
                in limited circumstances when no Personally Identifiable Information is 
                present or can be inferred.
```